src/wallet/KintoWalletFactory.sol

```diff
@@ -198,14 +198,16 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
     }
 
     /**
-     * @dev Send money to an account from privileged accounts
+     * @dev Send money to an account from privileged accounts or from kyc accounts to kyc accounts or contracts.
      * @param target The target address
      */
     function sendMoneyToAccount(address target) external payable override {
+        bool isPrivileged =
+            owner() == msg.sender || IAccessControl(address(kintoID)).hasRole(kintoID.KYC_PROVIDER_ROLE(), msg.sender);
+        require(isPrivileged || kintoID.isKYC(msg.sender), "KYC or Provider role required");
+        bool isContract = target.code.length > 0;
```

Comment:
```suggestion
        bool isValidTarget = kintoID.isKYC(target) || target.code.length > 0;
        require(targetIsKYCOrContract || isPrivileged, "Target is not valid");
```

i'd probably do the same as you did before (just for readability) and add a `isValidTarget` variable but up to you!

Reply:
and then remove lines 209 and 210